indexing-service/src/test/java/org/apache/druid/indexing/overlord/hrtr/HttpRemoteTaskRunnerTest.java

```diff
@@ -1606,6 +1607,63 @@ WorkerNodeService.DISCOVERY_SERVICE_KEY, new WorkerNodeService("ip1", 2, "0", Wo
     );
   }
 
+  @Test
+  public void testShutdown()
```

Comment:
Could you add some more info, either as a comment or in the method name itself to clarify what this method validates?

Reply:
Done